src/planwise/client/current_project/components/sidebar.cljs

```diff
@@ -15,6 +17,28 @@
    [:div.stat-title title]
    [:div.stat-value value]])
 
+(defn- route-by-tab-name [tab-name project-id]
+  (case tab-name
+    :demographics    (routes/project-demographics {:id project-id})
+    :facilities      (routes/project-facilities {:id project-id})
+    :transport       (routes/project-transport {:id project-id})))
+
+(defn- next-and-back-buttons [tab-name project-id]
+  (let [tabs [:demographics :facilities :transport]
```

Comment:
This ordering specification is duplicated. I think we need to rethink how we handle this. Probably move all this logic into the tabs definition.